src/Schemes/CoveredSchemes/Objects/Attributes.jl

```diff
@@ -178,7 +178,7 @@ end
   covering = (has_attribute(X, :simplified_covering) ? simplified_covering(X) : default_covering(X))
   for U in covering
     _, inc_sing = singular_locus(U)
-    D[U] = image_ideal(inc_sing)
+    D[U] = radical(image_ideal(inc_sing))
```

Comment:
Yes, the radical definitely makes sense. 

Just another question (not a todo for this PR, but a '#TODO' to put in the code to not forget it in the future: Should we also provide ideal_sheaf_of_singular_locus_reduced in the future? For many (most) applications -- like the one of @simonbrandhorst has in mind -- this would suffice and it internally avoids the full primary decomposition and relies on the significantly cheaper computation of minimal associated primes.

Reply:
So you mean that would be faster than `reduced_scheme(singular_locus(X))?` 
I don't like these long names for functions which do everything at once. Usually it is more user friendly to have fewer functions. (unless there is a significant speed difference)

Reply:
No, these are two different things:
- here the radical is applied to the ideal sheaf of the singular locus
  (I sure hope that the routines are aware that in each chart the radical has already been computed and hence just assign a flag instead of recomputing the radical!!!)
- the other thing is that in the computation of the singular locus, we can make use of the fact that we want the singular locus of the corresponding reduced scheme, if this is the case.

As I see it, you are not dealing with non-reduced schemes most of the time -- nor am I -- so we need a way to make use of this (bypassing the more expensive part of equidimensional_decomposition_weak and only using equidimensional_decomposition_radical in the internal decomposition inevitably happening inside singular_locus).